opencga-catalog/src/main/java/org/opencb/opencga/catalog/auth/authentication/CatalogAuthenticationManager.java

```diff
@@ -133,7 +133,8 @@ public OpenCGAResult resetPassword(String userId) throws CatalogException {
         String mailHost = this.emailConfig.getHost();
         String mailPort = this.emailConfig.getPort();
         try {
-            MailUtils.sendResetPasswordMail(email, newPassword, mailUser, mailPassword, mailHost, mailPort);
+            MailUtils.sendResetPasswordMail(email, newPassword, mailUser, mailPassword, mailHost, mailPort,
+                    "true", ((User) user.getResults().get(0)).getId());
```

Comment:
This hardcoded `ssl = "true"` should be in the configuration file. And should be a boolean instead of String.

Reply:
All the variables that are set in the code and are not in the configuration are necessary for the current version to work..... The objective of this task was to change the body of the email message. In the tests I discovered that it did not work with the current configuration, I have added those properties to make it work, if you want to bring that to the configuration it must be done in another requirement, which must be planned for the corresponding version because this must be released now.